libs/weshipit-ui/src/lib/button/button.tsx

```diff
@@ -5,18 +5,20 @@ export interface ButtonProps {
   accessoryLeft?: React.ReactNode;
   children?: React.ReactNode;
   style?: string;
+  href: string;
 }
 
 export function Button({
   variant,
   accessoryLeft,
   children,
   style,
+  href,
 }: ButtonProps) {
   if (variant === 'secondary') {
```

Comment:
variant should be a `switch` statement

we should not dupplicate the result `<a